pandas/core/missing.py

```diff
@@ -349,6 +351,13 @@ def _interpolate_scipy_wrapper(
             ) from err
     elif method == "akima":
         alt_methods["akima"] = _akima_interpolate
+    elif method == "cubicspline":
+        try:
```

Comment:
I don't believe this try except is necessary since CubicSpline was introduced in 0.18.0 and our minimum supported scipy version is 0.19

Reply:
OK, I agree, thanks! Using the same argument, I think that another PR could address the removal of the try-except block for the PCHIP interpolator above.

Reply:
Yup that should be okay to remove if that was introduced before 0.19.0

Reply:
OK, thanks, I will remove both then.